pkg/services/scm_provider/gitlab_test.go

```diff
@@ -0,0 +1,99 @@
+package scm_provider
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// func checkRateLimit(t *testing.T, err error) {
```

Comment:
Guessing you commented this because it would collide with the impl from github? Could rename that `checkGithubRateLimit` and this `checkGitlabRateLimit` since they might not use the same error messages? But "rate limit exceeded" is generic enough it might work for both by accident :D